src/cs3panel.tsx

```diff
@@ -89,18 +91,61 @@ export class Cs3HeaderWidget extends ReactWidget {
     }
 }
 
+export const Bottom = (props: BottomProps): JSX.Element => {
+
+    const [text, setText] = useState(props.message);
```

Comment:
Can't we initiate with empty string and remove this props.message?

Reply:
I can remove message from props, but if we initiate with empty string there will be no text on the bar after start